composer.json

```diff
@@ -2,43 +2,34 @@
     "name": "laravel/laravel",
     "type": "project",
     "description": "The Laravel Framework.",
-    "keywords": ["framework", "laravel"],
+    "keywords": [
+        "framework",
+        "laravel"
+    ],
     "license": "MIT",
     "require": {
         "php": "^8.2",
-        "archtechx/enums": "^0.3.1",
-        "bepsvpt/secure-headers": "^7.2",
-        "caneara/axiom": "^3.2",
-        "doctrine/dbal": "^3.4",
+        "bepsvpt/secure-headers": "^7.4",
         "guzzlehttp/guzzle": "^7.2",
-        "illuminate/console": "^9.0",
         "itsgoingd/clockwork": "^5.1",
-        "kreait/laravel-firebase": "^5.0",
-        "laravel/framework": "^9.0",
-        "laravel/sanctum": "^3.0",
-        "laravel/slack-notification-channel": "^2.3",
-        "laravel/tinker": "^2.5",
-        "league/flysystem-aws-s3-v3": "^3.3",
-        "psr/log": "~2",
-        "rollbar/rollbar-laravel": "^7.2",
-        "spatie/eloquent-sortable": "^4.0",
```

Comment:
As it was not being used, I removed it and added to the Package Recommendations section of the README